doc/build-unix.md

```diff
@@ -22,19 +21,20 @@ distributions](#linux-distribution-specific-instructions), or the
 
 C++ compilers are memory-hungry. It is recommended to have at least 1.5 GB of
 memory available when compiling Bitcoin Core. On systems with less, gcc can be
-tuned to conserve memory with additional CXXFLAGS:
+tuned to conserve memory with additional `CMAKE_CXX_FLAGS`:
 
 
-    ./configure CXXFLAGS="--param ggc-min-expand=1 --param ggc-min-heapsize=32768"
+    cmake -B build -DCMAKE_CXX_FLAGS="--param ggc-min-expand=1 --param ggc-min-heapsize=32768"
 
-Alternatively, or in addition, debugging information can be skipped for compilation. The default compile flags are
-`-g -O2`, and can be changed with:
+Alternatively, or in addition, debugging information can be skipped for compilation.
+For the default build type `RelWithDebInfo`, the default compile flags are
+`-O2 -g`, and can be changed with:
 
-    ./configure CXXFLAGS="-g0"
+    cmake -B build -DCMAKE_CXX_FLAGS_RELWITHDEBINFO="-O2 -g0"
 
 Finally, clang (often less resource hungry) can be used instead of gcc, which is used by default:
 
-    ./configure CXX=clang++ CC=clang
+    cmake -B build -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CC_COMPILER=clang
```

Comment:
Should this say:
`cmake -B build -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang`
instead?

When I follow these instructions as written, I get a Cmake Warning that the CMAKE_CC_COMPILER variable was not used by the project.

Reply:
Indeed, that's what we have in other cases: https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md?plain=1#L11

Reply:
Thanks! Fixed in https://github.com/bitcoin/bitcoin/pull/30744.